src/vals.c

```diff
@@ -15,22 +15,12 @@ struct rill_packed vals
 
 typedef struct htable vals_rev_t;
 
-static size_t vals_cap(struct vals *vals)
-{
-    return sizeof(*vals) + vals->len * sizeof(vals->data[0]);
-}
-
-static rill_val_t vals_itov(struct vals *vals, size_t index)
-{
-    assert(index <= vals->len);
-    return vals->data[index - 1];
-}
-
 static size_t vals_vtoi(vals_rev_t *rev, rill_val_t val)
 {
     if (!val) return 0; // \todo giant hack for coder_finish
 
     struct htable_ret ret = htable_get(rev, val);
+    if (!ret.ok) { printf("checkvals: find: %zu\n", val); }
```

Comment:
This seems like debug code which I would remove.

Reply:
:'( oops! fixed